test/server-spec.js

```diff
@@ -999,6 +1000,30 @@ describe('Server manages requests from advanced privileged users in a set up rep
                 done();
             });
     });
+
+    it('reacts to repository renames', function testRepoRename(done) {
+        const body = {
+            "action": "renamed",
+            "changes": {
+              "repository": {
+                "name": {
+                  "from": testExistingRepo.name
+                }
+              }
+            },
+            "repository": {
+              "name": renamedRepo,
+              "full_name": `${testExistingRepo.owner}/${renamedRepo}`,
+              "owner": {
+                "login": testExistingRepo.owner
+              }
+            }
+        };
+        req.post('/' + config.hookPath)
+            .send(body)
+            .set('X-Github-Event', 'repository')
+            .expect(200, done);
```

Comment:
Shouldn't this also test that the stored repo and its associated PRs have been updated accordingly?

Reply:
The [`after` function](https://github.com/w3c/ash-nazg/pull/223/files#diff-1b1c51bec6088eb4bb95466d455303cb6e8b43f422114b6d6fc88b0a68e43f69R568) will clean the PRs. If the PR with the new name doesn't exist, the cleanup will fail.
That being said, I was wondering if we shouldn't separate these tests because the new one is relying on PRs and repo created in previous tests. This also makes it harder to test the repository transfer. WDYT?

Reply:
ah, got it wrt `after`, that makes sense. Maybe add a comment to that effect on line 1025?

Let's merge this, and file an issue (or a PR :) to clean up how the test suite is structured.

Reply:
I just added a comment to explain how the renamed is checked.
I'll merge now and will prepare separate PRs for the script to update the existing webhooks and to separate the tests so we can also check the repo transfer.